### PR TITLE
add additional method for stats

### PR DESCRIPTION
**Pull request by nvmdava**

### CODE REVIEW

File: src/main/java/seedu/address/model/customer/Customer.java [hunk 1]
```diff
@@ -63,6 +64,42 @@ public Optional<Address> getAddress() {
...
+     * Get total number of active commissions made by this customer
+     * @return number of commissions
+     */
+    public long getActiveCommissionCount() {
+        return commissions.getActiveSize();
+    }
+
+    /**
+     * Get last date of customer's commissions
+     * @return
```
Comment (chloeelim) [inline src/main/java/seedu/address/model/customer/Customer.java:97]: The @return tag here is empty- is this intentional? I think we should either fill it in or delete the tag.

**Review by chloeelim — COMMENTED** [review]
Overall L(almost)GTM, just some very minor nitpicks with the JavaDocs, I think according to the style guide, adding punctuation to your JavaDoc headers and params will be good. There's also an empty @return tag in your JavaDoc.

File: src/main/java/seedu/address/model/commission/UniqueCommissionList.java [hunk 2]
```diff
@@ -49,6 +50,39 @@ public void add(Commission toAdd) {
...
 
+    /**
+     * Returns the size of the commission list.
+     */
+    public int getSize() {
+        return internalList.size();
+    }
+
+    /**
+     * Returns the size of active commissions.
```
Comment (ngxingyu) [inline src/main/java/seedu/address/model/commission/UniqueCommissionList.java:61]:
```suggestion
     * Returns the number of active commissions.
```

File: src/main/java/seedu/address/model/commission/UniqueCommissionList.java [hunk 3]
```diff
@@ -49,6 +50,39 @@ public void add(Commission toAdd) {
...
+     */
+    public long getActiveSize() {
+        return internalList.stream()
+                .filter(commission -> !commission.getCompletionStatus().isCompleted)
+                .count();
+    }
+
+    /**
+     * Get Last Deadline.
```
Comment (ngxingyu) [inline src/main/java/seedu/address/model/commission/UniqueCommissionList.java:70]:
```suggestion
     * Returns the last Deadline.
```

File: src/main/java/seedu/address/model/customer/Customer.java [hunk 4]
```diff
@@ -63,6 +64,38 @@ public Optional<Address> getAddress() {
...
+
+    /**
+     * Get total number of commission made by this customer.
+     */
+    public long getCommissionCount() {
+        return commissions.getSize();
+    }
+
+    /**
+     * Get total number of active commissions made by this customer.
```
Comment (ngxingyu) [inline src/main/java/seedu/address/model/customer/Customer.java:86]:
```suggestion
     * Returns the total number of active commissions made by this customer.
```

File: src/main/java/seedu/address/model/customer/Customer.java [hunk 5]
```diff
@@ -63,6 +64,38 @@ public Optional<Address> getAddress() {
...
+    public double getRevenue() {
+        double revenue = 0;
+        for (Commission commission : commissions) {
+            revenue += commission.getFee().fee;
+        }
+        return revenue;
+    }
+
+    /**
+     * Get total number of commission made by this customer.
```
Comment (ngxingyu) [inline src/main/java/seedu/address/model/customer/Customer.java:79]:
```suggestion
     * Returns the total number of commissions made by this customer.
```

File: src/main/java/seedu/address/model/customer/Customer.java [hunk 6]
```diff
@@ -63,6 +64,38 @@ public Optional<Address> getAddress() {
         return Optional.ofNullable(address);
     }
 
+    /**
+     * Get total revenue generated from the customer.
```
Comment (ngxingyu) [inline src/main/java/seedu/address/model/customer/Customer.java:68]:
```suggestion
     * Returns total revenue generated from the customer.
```

**Review by ngxingyu — COMMENTED** [review]
More minor javadocs changes (mainly starting with plural), but the rest of it looks fine.

**Comment by nvmdava** [comment]
??????????????????????????????????????????

**Review by ngxingyu — APPROVED** [review]
LGTM!

**Pull request merged**